web/src/components/serviceFleetSection.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import ServiceFleetCard from './serviceFleetCard';
+
+export default function ServiceFleetSection({ products, productTitle }) {
+    return (
+        <section className="container my-14">
+            <h2 className="font-bold text-4xl text-brand-1 text-center mb-12">
+                {productTitle}
+            </h2>
+            <div className="flex flex-col items-center md:flex-row md:flex-wrap justify-center gap-9">
```

Comment:
I added a screenshot, thinking grid might be better here.

Reply:
@asasmith i tried both a few times and ultimately the flex looked better across browser sizes. Grid smooshed the cards together, event when I tried to add all the different breakpoints

Reply:
@asasmith I am sure you saw but updated to grid, lmk if youre happy with the way it looks across browser sizes

Reply:
yeah I think this is better. Left a comment/question about the routing above.